TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.cpp

```diff
@@ -1016,3 +1047,104 @@ bool Moveable::MeshExists(int index) const
 
 	return true;
 }
+
+//Attach camera and camera target to a mesh of an object.
+void  Moveable::AttachCamera(short camMeshId, short targetMeshId)
+{
+	ObjCamera(m_item, camMeshId, m_item, targetMeshId, true);
+}
+
+//Lara will undraw her weapons if they are drawn, throw away flare if she helds one in her hand.
+void   Moveable::UndrawWeapons()
+{
+	if (m_item->IsLara())
```

Comment:
You should avoid wrapping whole funciton body into a condition. Instead, according to #294 , you should do early exit from function:

` if (!m_item->IsLara())`
`     return;`

Reply:
Changed to make it always Lara Item. No check needed anymore :)